src/main/java/com/sourcegraph/webhook/EventSerializer.java

```diff
@@ -36,6 +50,17 @@ private static JsonElement render(Object o) {
     private static Adapter<BuildStatusSetEvent> BuildStatusSetEventAdapter = ((element, event) -> {
         element.addProperty("commit", event.getCommitId());
         element.add("status", render(event.getBuildStatus()));
+
+        // Find Pull Requests
+        PullRequestCommitSearchRequest searchRequest = new PullRequestCommitSearchRequest.Builder(event.getCommitId()).build();
+        PageRequest pageRequest = new PageRequestImpl(0,100);
+        Page<PullRequest> prs = pullRequestService.searchByCommit(searchRequest, pageRequest);
+
+        JsonArray ja = new JsonArray();
+        prs.stream().forEach(pr -> ja.add(render(pr)));
+        if (ja.size() > 0) {
+            element.add("pullRequests", ja);
+        }
```

Comment:
Why only add the `pullRequests` array if it has elements?

As a JSON consumer I'd expect properties to always be there, especially when it's an array, because I'd rather do the `len(event.pullRequests)` check on my end than check whether it's nil or not.